CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.2)
 project(MorphIO VERSION 2.0.0)
 
+add_subdirectory(3rdparty/coding-conventions/cpp)
+bob_begin_cxx_flags()
```

Comment:
@tristan0x Is this how it is supposed to be called ?

Reply:
`bob.cmake` is a bit special, and not ready to be widely used.
It is completely incompatible with ` CMAKE_BUILD_TYPE`. I do not advise you to use it right for now. Sorry!